Update Notify RadarrSonarr to include Lidarr

martinkeat opened the pull request:
New folder created with update of notify Sonarr or Radarr plugin. Now includes Lidarr for audio.

martinkeat committed:
Update Notify RadarrSonarr

New folder created with update of notify Sonarr or Radarr plugin. Now includes Lidarr for audio.

supersnellehenk commented:
The typescript version is missing :)

supersnellehenk commented on FlowPlugins/CommunityFlowPlugins/tools/notifySonarrRadarrLidarr/1.0.1/index.js:
```diff
@@ -0,0 +1,142 @@
+            case 6:
+                if (!(arr === 'lidarr')) return [3 /*break*/, 9];
+                args.jobLog('Refreshing Lidarr...');
+                artistId = 'artistId'; // Placeholder for artist ID
```
Does this even work? Radarr & Sonarr need the IDs, a placeholder for Lidarr seems a bit odd.

HaveAGitGat committed:
Merge branch 'master' into pr/595

HaveAGitGat committed:
Add Lidarr support to notifyRadarrOrSonarr 2.0.0

Extend the existing plugin to support Lidarr alongside Radarr and Sonarr
instead of adding a separate notifySonarrRadarrLidarr plugin. Uses Lidarr
/api/v1 with RefreshArtist and artistIds. Skips the imdb lookup path for
Lidarr since music releases do not carry imdb ids.

HaveAGitGat committed:
Merge branch 'master' into master
